tests/unit/lms/security_test.py

```diff
@@ -506,6 +510,20 @@ def test_it_returns_LTIUsers_from_OAuth2_state_params(
         canvas_oauth_callback_schema.lti_user.assert_called_once_with()
         assert lti_user == canvas_oauth_callback_schema.lti_user.return_value
 
+    def test_it_returns_LTIUser_from_openid_auth_schema(
+        self,
+        LTI13AuthSchema,
+        lti13_auth_schema,
+        pyramid_request,
```

Comment:
Worth trying deleting the trailing comma and running black. It might back it up onto one line?